visualization/app/codeCharta/ui/nodeContextMenu/nodeContextMenu.component.spec.ts

```diff
@@ -441,6 +441,15 @@ describe("nodeContextMenuController", () => {
 
 			expect(dialogService.showErrorDialog).toBeCalled()
 		})
+
+		it("should prevent duplicate blacklist object regarding issue #2419", () => {
```

Comment:
It is not necessary to refer to the issue number. The reference is placed in the commit message anyway. Furthermore, I don't want to review the issue to be able to understand the test case :-)

Reply:
You are correct that the code references the issue. The test itself does not and if I want to know more about that, it's easier to identify the corresponding issue from a failing test case. :-)
Having self explanatory tests is always ideal. For regression tests, it's often still useful to know why a test case exists.

Reply:
The test case is well described, so I am fine with keeping the (helpful) reference, of course.
It's just a personal preference.